custom/conf/app.ini.sample

```diff
@@ -185,6 +185,10 @@ STATIC_ROOT_PATH =
 APP_DATA_PATH = data
 ; Application level GZIP support
 ENABLE_GZIP = false
+; Application profiling (memory and cpu)
+; For "web" command it listens on localhost:6060
+; For "serve" command it dumps to disk at $TMPDIR/gitea_serve_pprof_<username>_id
```

Comment:
Is it possible to define this path as a setting? If not that's fine.

Reply:
Thats a nice addition, I will name it `PPROF_FILE_PATH`. Good? Should we only allow absolute paths or also expand e.g $TMPDIR ? Absolute paths are much easier to implement.

Reply:
it should be treated same as other paths in Gitea